site/content/internal/infrastructure/kubernetes/kubernetes-troubleshooting.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Kubernetes Troubleshooting"
```

Comment:
Is this page for maintenance of for troubleshooting?

Reply:
yes